test/jdk/java/awt/print/Dialog/PrintDlgPageable.java

```diff
@@ -25,105 +25,118 @@
+
+        // Simply draw two rectangles
+        Graphics2D g2 = (Graphics2D) g;
+        g2.setColor(Color.black);
+        g2.translate(pf.getImageableX(), pf.getImageableY());
+        g2.drawRect(1, 1, 200, 300);
+        g2.drawRect(1, 1, 25, 25);
+        PassFailJFrame.log("print method called " + pi + " Orientation " + pf.getOrientation());
+        return Printable.PAGE_EXISTS;
+    }
 }
 
 class PageableHandler implements Pageable {
 
-        PageFormat pf = new PageFormat();
+    PageFormat pf = new PageFormat();
 
-        public int getNumberOfPages() {
-                return PrintDlgPageable.arg;
-                //return 0;
-        }
+    public int getNumberOfPages() {
+        return PrintDlgPageable.arg;
+    }
 
-        public Printable getPrintable(int pageIndex) {
-                return new PrintDlgPageable();
-        }
+    public Printable getPrintable(int pageIndex) {
+        return new PrintDlgPageable();
+    }
 
-        public PageFormat getPageFormat(int pageIndex) {
-                System.out.println("getPageFormat called "+pageIndex);
-                if (pageIndex == 0) {
-                        pf.setOrientation(PageFormat.PORTRAIT);
-                        System.out.println("Orientation returned from Pageable "+findOrientation(pf.getOrientation()));
-                        return pf;
-                } else {
-                        pf.setOrientation(PageFormat.LANDSCAPE);
-                        System.out.println("Orientation returned from Pageable "+findOrientation(pf.getOrientation()));
-                        return pf;
-                }
+    public PageFormat getPageFormat(int pageIndex) {
+        System.out.println("getPageFormat called " + pageIndex);
+        if (pageIndex == 0) {
+            pf.setOrientation(PageFormat.PORTRAIT);
+            PassFailJFrame.log("Orientation returned from Pageable " + findOrientation(pf.getOrientation()));
+            return pf;
+        } else {
+            pf.setOrientation(PageFormat.LANDSCAPE);
+            PassFailJFrame.log("Orientation returned from Pageable " + findOrientation(pf.getOrientation()));
+            return pf;
         }
+    }
 
-        public String findOrientation(int orient) {
-                if (orient == PageFormat.LANDSCAPE) {
-                        return "LANDSCAPE";
-                }else if (orient == PageFormat.PORTRAIT) {
-                        return "PORTRAIT";
-                } else if (orient == PageFormat.REVERSE_LANDSCAPE) {
-                        return "REVERSE LANDSCAPE";
-                } else {
-                        return null;
-                }
+    public String findOrientation(int orient) {
+        if (orient == PageFormat.LANDSCAPE) {
+            return "LANDSCAPE";
+        } else if (orient == PageFormat.PORTRAIT) {
+            return "PORTRAIT";
+        } else if (orient == PageFormat.REVERSE_LANDSCAPE) {
+            return "REVERSE LANDSCAPE";
```

Comment:
Is it required to check for `REVERSE_LANDSCAPE `as page formatting is set only either `LANDSCAPE `or `PORTRAIT `?